scripts/loadEmptyPrefsSets.js

```diff
@@ -0,0 +1,127 @@
+/*!
+Copyright 2018 Raising the Floor US
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://github.com/GPII/universal/blob/master/LICENSE.txt
+*/
+
+// This script generates unique pref sets and loads them into CouchDB.
+//
+// See this JIRA for more information: https://issues.gpii.net/browse/GPII-3381
+// The options to this script must be set as environment variables when calling it,
+// these are:
+//   * COUCHDB_URL: This neeeds to be in the form: "http://user:pass@host:port/dbname"
+//   * NEW_USERS: The amount of new empty preferences sets you want to create
+//
+
+"use strict";
+
+var process = require("process"),
+    fluid = require("infusion"),
+    gpii = fluid.registerNamespace("gpii"),
+    uuid = uuid || require("node-uuid");
+
+require("gpii-pouchdb");
+
+var emptyPrefsSetBlock = {
+    "flat": {
+        "contexts": {
+            "gpii-default": {
+                "name": "Default preferences",
+                "preferences": {}
+            }
+        }
+    }
+};
+
+fluid.defaults("gpii.uuidLoader", {
+    gradeNames: ["gpii.pouch"],
+    dbOptions: {
+        //name: "http://localhost:5984/gpii"
+        name: process.env.COUCHDB_URL
+    },
+    invokers: {
+        generatePrefsSet: { funcName: "gpii.uuidLoader.generatePrefsSet" },
+        addPrefsSet: {
+            funcName: "gpii.uuidLoader.addPrefsSet",
+            args: [ "{that}", "{arguments}.0", "{arguments}.1"]
+        }
+    }
+});
+
+gpii.uuidLoader.generatePrefsSet = function () {
+    var gpiiKey = uuid.v4();
+    var currentTime = new Date().toISOString();
```

Comment:
The logic from line 57-83 is similar to [convertPrefs.js](https://github.com/GPII/universal/blob/master/scripts/convertPrefs.js#L43-L70). Can you create a shared function for it so the data structure generation only needs to be maintained at one place? Thanks.